website/sidebars.js

```diff
@@ -49,8 +57,11 @@ module.exports = {
         },
       ],
       Advanced: ['overview/beanstalk/beanstalk'],
+      Tutorials: fbContent({
+        internal: 'overview/tutorials/tutorials',
```

Comment:
:D we need to wrap the string with an array, otherwise docusaurus is going to raise a typeerror, i.e.
```jsx
internal: ['overview/tutorials/tutorials'],
```

(you can replicate the error by swapping `internal` and `external` entries)